common/membership/rpMonitor.go

```diff
@@ -238,6 +245,7 @@ func fetchCurrentBootstrapHostports(manager persistence.ClusterMetadataManager,
 
 	for {
 		resp, err := manager.GetClusterMembers(
+			context.Background(),
```

Comment:
why not TODO()?

Reply:
Good catch. Missed this one when converting `Background` to `TODO` in this file. Updated to `TODO`